### DISCUSSION TITLE
Plug-in to convert images

### DISCUSSION THREAD

**Discussion by lduperval** [doublecmd/doublecmd#1024] [Suggestions]
Hi,

Directory Opus has a cool feature that allows you to select an image or a number of images and converts them or resizes them.

Is there a way to do something like this in DoubleCommander? 

So you select a file, right click, select convert, choose your options (file format, size, rotate, other?) and click go. Then the file(s) are converted according to the chosen options.

Has anyone built something like this or can I use an existing plugin to see how I could implement it?

Thanks,

L

**Comment by alexx2000** [comment]
Hi, see [Graphic Converter](https://totalcmd.net/plugring/graphicconverter.html)

**Reply by lduperval** [reply]
Thanks. Is there a page somewhere that explains how to install a plugin for someone who has never installed a DC plugin?

I tried extracting the zip file in the DC plugins folder, hoping the "automatic install" would work but nothing happened.

So I added the wcx64 file manually using the DC options menu and associated some extensions with it, but I still don't see a difference.

Thanks,

L

**Comment by alexx2000** [comment]
Plugin `Readme.txt` contains info about usage:

> Select graphic files in TC, then select Files-Pack..., in combo box in Packer section panel select GraphConv.
> You may change output file settings by pressing Configure button.

**Comment by lduperval** [comment]
Hi,

Thanks for this. In the Packer combo box, I don't see GraphConv.

In the Readme it says:

TC has "auto install plugin" feature. Just enter to archive with plugin and TC ask you about plugin installing. 

However, I am on Double Commander, not Total Commander. So when I enter the Zip file, it uses 7-zip and I don't see the auto install. That's why I was asking about installing.

I'll try to install TotalCommander to see if I get better results.

**Comment by alexx2000** [comment]
Hi,

You wrote that you installed a plugin and associate it with some extension. So in the pack dialog you should choose this extension.

https://doublecmd.github.io/doc/en/configuration.html#ConfigPlugins

**Comment by lduperval** [comment]
Hi,

Yes, I did say I installed it. I'm just not convinced I did it correctly. In the options window, this is what I have for the plugin:

%commander_path%\plugins\wcx\wcx_GraphicConverter_1.8.7.1\GraphicConverter.wcx

If I select the jpg from the dropdown and click configure, I get the error:

> Cannot load WCX module %commander_path%\plugins\wcx\wcx_GraphicConverter_1.8.7.1\GraphicConverter.wcx

So I think it was not installed correctly. But I'm not sure.

L
